src/client/pfs/pfs.proto

```diff
@@ -484,6 +484,18 @@ message FsckResponse {
   string error = 2;
 }
 
+// Messages specific to the new storage layer.
+
+message PutTarRequest {
+  Commit commit = 1;
+  bytes data = 2;
+}
+
+message GetTarRequest {
+  Commit commit = 1;
+  string glob = 2;
```

Comment:
I think it'd match a little better with our other RPCs if this was expressed as a `File` in which the `Path` could be a glob pattern, that's how `GlobFile` and `GetFile` presently work.

Reply:
Done. We discussed this decision offline and decided to use the `File` type to maintain consistency.